manifest.md

```diff
@@ -159,8 +160,8 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 
     Each item in the array MUST be a [descriptor](descriptor.md).
     The array MUST have the base image at index 0.
-    Subsequent layers MUST then follow in the order in which they are to be layered on top of each other.
-    The algorithm to create the final unpacked filesystem layout MUST be to first unpack the layer at index 0, then index 1, and so on.
+    Subsequent layers MUST then follow in stack order.
+    The final filesystem layout MUST be from [applying](layer.md#applying) the layers in stack order (i.e. from layers[0] to layers[len(layers)]).
```

Comment:
I still think we need to specify something about the initial state before the layers are applied (#318).  I also [changed “be” to “match” in that PR to allow for implementations like union mounting][1].

Also backticks around the `layers[…]` stuff.

[1]: https://github.com/opencontainers/image-spec/pull/318#issue-177482014